config.js

```diff
@@ -29,7 +29,14 @@ module.exports = {
     },
     ntco: {
       action: [
-        'awaiting-endorsement'
+        'awaiting-endorsement',
+        'resolved'
+      ]
+    },
+    outgoingNtco: {
```

Comment:
What does the notification they get sent actually say?

My guess is that it will need to have custom content as they won't have any visibility of the task in question.